<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QSplitter" name="splitter_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QLabel" name="lb_inFormat">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Имя файла с входными данными</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="le_inFormat"/>
            <widget class="QComboBox" name="cb_inFormat">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <item>
              <property name="text">
               <string>.bin</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>.txt</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>.ini</string>
              </property>
             </item>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QSplitter" name="splitter_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QLabel" name="lb_outFormat">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Имя файла для сохранения</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="le_outFormat"/>
            <widget class="QComboBox" name="cb_outFormat">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <item>
              <property name="text">
               <string>.bin</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>.txt</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>.ini</string>
              </property>
             </item>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QSplitter" name="splitter_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QLabel" name="lb_bit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>8 Бит</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="le_bit"/>
           </widget>
          </item>
          <item>
           <widget class="QSplitter" name="splitter_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QLabel" name="lb_timer">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Вермя таймера</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="le_timer"/>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QSplitter" name="splitter">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <widget class="QCheckBox" name="chb_startTimer">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Запуск по таймеру</string>
             </property>
            </widget>
            <widget class="QCheckBox" name="chb_clearInData">
             <property name="text">
              <string>Очистить входной файл</string>
             </property>
            </widget>
            <widget class="QPushButton" name="pb_start">
             <property name="text">
              <string>Старт</string>
             </property>
            </widget>
            <widget class="QPushButton" name="pb_stop">
             <property name="text">
              <string>Стоп</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>669</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
